conf/repos.yaml.template

```
@@ -198,3 +198,7 @@ REPOS:
   GR_YUM_REPO:
     URL:
     GPG_URL:
+
+  PYTHON:
+    SYNC:
+      URL: 'https://pypi.org/project/pytest'
```

Would it make sense to split it by potential sources and/or list of repos to work with? I thought something like
```suggestion
  PYTHON:
    PYPI:
      URL: 'https://pypi.org/project/'
      PROJECTS:
      - pytest
      - wrapanapi
    PULP_FIXTURES:
      URL: 'https://fixtures.pulpproject.org/'
      PROJECTS:
      - python-pypi
```

But for now I think we can start just with 
```suggestion
  PYTHON:
    PYPI:
      URL: 'https://pypi.org/project/pytest'
```
and expand later when needed.

---

That makes sense. Adapted it.